src/verifai/falsifier.py

```diff
@@ -1,14 +1,26 @@
 from abc import ABC
-from verifai.server import Server
+from verifai.server import Server, ParallelServer
+from verifai.scenic_server import ScenicServer, ParallelScenicServer
 from verifai.samplers import TerminationException
 from dotmap import DotMap
-from verifai.monitor import mtl_specification, specification_monitor
+from verifai.monitor import mtl_specification, specification_monitor, multi_objective_monitor
 from verifai.error_table import error_table
 import numpy as np
+import pickle
+import progressbar
+import ray
+from statsmodels.stats.proportion import proportion_confint
+import time
+
+def parallelized(server_class):
+    if server_class == Server:
+        return ParallelServer
+    elif server_class == ScenicServer:
+        return ParallelScenicServer
 
 class falsifier(ABC):
     def __init__(self, monitor, sampler_type=None, sampler=None, sample_space=None,
-                 falsifier_params=None, server_options={}, server_class=Server):
+                 falsifier_params=None, server_options={}, server_class=Server, use_carla=False):
```

Comment:
The argument `use_carla` appears to be unused -- in any case obviously the generic falsifier class should not have any simulator-specific arguments (communication with the simulator is the job of the Server).